docs/Deploying-Pega-on-AKS.md

```diff
@@ -461,6 +455,8 @@ Configure the following parameters so the pega.yaml Helm chart matches your depl
 
 - Access your Azure SQL database.
 
+- Access to the ElasticSearch service deployed using SRS nodes.
```

Comment:
slight tweak to make it parallel and show SRS is recommended but optional>>
Access your ElasticSearch service (For 8.6 and later, Pega recommends deploying your service using an SRS cluster).

Reply:
done.